release-notes/7.0/preview/api-diff/rc1/Microsoft.NETCore.App/7.0-rc1_System.Security.Cryptography.md

```diff
@@ -0,0 +1,125 @@
+# System.Security.Cryptography
+
+``` diff
+ namespace System.Security.Cryptography {
+     public abstract class Aes : SymmetricAlgorithm {
+-        public static new Aes Create();
++        [UnsupportedOSPlatformAttribute("browser")]
++        public static new Aes Create();
+     }
+     [ObsoleteAttribute("Derived cryptographic types are obsolete. Use the Create method on the base type instead.", DiagnosticId="SYSLIB0021", UrlFormat="https://aka.ms/dotnet-warnings/{0}")]
+     public sealed class AesCryptoServiceProvider : Aes {
+-        public AesCryptoServiceProvider();
++        [UnsupportedOSPlatformAttribute("browser")]
++        public AesCryptoServiceProvider();
+     }
+-    [ObsoleteAttribute("Derived cryptographic types are obsolete. Use the Create method on the base type instead.", DiagnosticId="SYSLIB0021", UrlFormat="https://aka.ms/dotnet-warnings/{0}")]
+-    public sealed class AesManaged : Aes
++    [ObsoleteAttribute("Derived cryptographic types are obsolete. Use the Create method on the base type instead.", DiagnosticId="SYSLIB0021", UrlFormat="https://aka.ms/dotnet-warnings/{0}")]
++    [UnsupportedOSPlatformAttribute("browser")]
++    public sealed class AesManaged : Aes
```

Comment:
This looks broken. Should this at least have a `{}` when no members are changing?

Reply:
Opened issue https://github.com/dotnet/arcade/issues/10981

Reply:
And will fix manually